src/Service/StatisticService.php

```diff
@@ -44,4 +44,29 @@ public function getPageVisitsCountByPicture(Picture $picture): int
     {
         return $this->pageVisitRepository->count(['picture' => $picture]);
     }
+
+
+    public function getPageVisitsCountByRouteWithDates(string $routeName): array
+    {
+        return $this->pageVisitRepository->createQueryBuilder('pv')
+        ->select("DATE_FORMAT(pv.visitedAt, '%Y-%m-%d') AS date, COUNT(pv.id) AS count")
```

Comment:
No. It's not the role of the Model to format a date. So don't use this DATE_FORMAT() thing.